raft/log.go

```diff
@@ -381,6 +381,11 @@ func (l *Log) Open(path string) error {
 		}
 		l.config = c
 
+		// If only node in cluster, then promote to leader immediately.
+		if len(l.config.Nodes) == 1 {
+			l.leaderID = l.id
```

Comment:
To be more specific -- I see this being done during initialize, and it caught my eye. Perhaps this attribute needs to be set as well.